bower.json

```diff
@@ -7,6 +7,6 @@
     "o-colors": "^4.2.2",
     "o-icons": ">=4.5.0 <6",
     "o-normalise": "^1.2.0",
-    "o-brand": ">=2.2.0 <4"
+    "o-brand": "^3.1.0"
```

Comment:
The fact that this will no longer accept o-brand version 2 would make this a breaking change, is that correct?

Reply:
As we said, this time around it isn't because it is an internal (to Origami) component that only we use. It won't be the case further down the line,  but it warrants an explanation for our breaking changes in the documentation. 👇 
https://github.com/Financial-Times/origami/issues/17

Reply:
Just for the paper trail: I thought this was the case too but unfortunately not. The build for `o-header@7.5.2` is failing as it requires `"o-brand": "^2.0.2"` and `"o-buttons": "^5.0.0"`, so whilst we can current update majors of o-brand silently behind the scenes we can't remove the range without a breaking. Fortunately as most users include components using a caret `o-header@^7.5.2` which has reduced the impact.